scenario/README.md

```diff
@@ -0,0 +1,108 @@
+# Scenarios
+
+**Inductiva API** provides a framework for users to create personalized scenarios. Scenarios describe a physical model of a process in nature, that can be simulated. With a scenario, the complexity of configuring the simulators is abstracted and allows users to configure only a few parameters to make an exploration of the physical model.
```

Comment:
the underlying simulators is abstracted away and this allows users...


You also need to explain that scenarios are python classes.